polling/openexchangerates.go

```diff
@@ -112,7 +112,7 @@ func CallOpenExchangeRates(c *config.Config) (response OpenExchangeRatesResponse
 	// Price is inverted
 	if err == nil {
 		for k, v := range OpenExchangeRatesResponse.Rates {
-			OpenExchangeRatesResponse.Rates[k] = 1 / v
+			OpenExchangeRatesResponse.Rates[k] = v
```

Comment:
the whole loop can actually be removed now

Reply:
Removed it, as long as the api layer one too